### PR TITLE
Rewording notification reasons

### PR DESCRIPTION
**Pull request by dirathea**
> 🦄
Rewording notification reasons and add new reasons based on https://developer.github.com/v3/activity/notifications/#notification-reasons.

Issues : https://github.com/sindresorhus/notifier-for-github/issues/100

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#100: Weird notification reasons messages](https://issuehunt.io/repos/18894771/issues/100)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

File: source/lib/defaults.js [hunk 1]
```diff
@@ -2,11 +2,14 @@ export const notificationReasons = new Map([
...
-	['team_mention', 'Your team was mentioned'],
+	['comment', 'You commented on this thread'],
+	['mention', 'New updates from thread that you were mentioned'],
+	['team_mention', 'New updates from thread that your team was mentioned'],
 	['state_change', 'Thread status changed'],
-	['assign', 'You were assigned to the issue']
+	['assign', 'You were assigned to the thread'],
+	['security_alert', 'New security vulnerability found on this repository'],
+	['invitation', 'You accepted an invitation to contribute to this repository.'],
+	['review_requested', `You, or a team you're a member of, were requested to review a pull request.`]
```
Comment (notlmn) [inline source/lib/defaults.js]:
```suggestion
	['review_requested', 'You, or a team you're a member of, were requested to review a pull request.']
```

**Review by notlmn — CHANGES_REQUESTED** [review]
Some of these reasons are too long to be displayed on a notification, they would be condensed, losing the point.

It's better if you write some condensed reasons instead of copy pasting.

**Comment by dirathea** [comment]
@notlmn is there any updates on this?

**Comment by notlmn** [comment]
Maybe you should also consider resolving https://github.com/sindresorhus/notifier-for-github/issues/100#issuecomment-558985871 in the same PR.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
> Maybe you should also consider resolving [#100 (comment)](https://github.com/sindresorhus/notifier-for-github/issues/100#issuecomment-558985871) in the same PR.

I've never seen this in my notifications? is it hidden in the chrome extension?

**Comment by notlmn** [comment]
> I've never seen this in my notifications? is it hidden in the chrome extension?

It's referring the way notification count badge is displayed in the browser UI. The 'reasons' for notifications are displayed in Chrome but not in Firefox, that's a limitation from the browser.

**Comment by dirathea** [comment]
Ah so that issue should be related with my PR, Will check

**Comment by dirathea** [comment]
I just update the PR related to https://github.com/sindresorhus/notifier-for-github/issues/100#issuecomment-558985871. I think the issue is because unhandled error on fetch process, and I classify this error as `network error`. What do you think @notlmn ?

File: source/lib/defaults.js [hunk 2]
```diff
@@ -2,17 +2,21 @@ export const notificationReasons = new Map([
...
+	['security_alert', 'New security vulnerability found'],
+	['invitation', 'You accepted an invitation'],
+	['review_requested', 'PR Review Requested']
 ]);
 
 export const errorTitles = new Map([
 	['missing token', 'Missing access token, please create one and enter it in Options'],
 	['server error', 'You have to be connected to the Internet'],
 	['client error', 'Invalid token, enter a valid one'],
+	['network error', 'Failed to fetch Github Notification API'],
```
Comment (notlmn) [inline source/lib/defaults.js]: Messages for `'network error'` and `'server error'` should be exchanged here.

And for server errors make it to something like "GitHub having issues serving requests"? Feel free to change it to something relevant.

**Review by notlmn — CHANGES_REQUESTED** [review]

File: source/lib/defaults.js (hunk 2 above)
Comment (dirathea) [reply source/lib/defaults.js]: Agreed, will do. Thanks

**Review by notlmn — APPROVED** [review]
Looks good!

**Comment by notlmn** [comment]
Thanks for the PR @dirathea!

**Pull request merged**